demos/serverless/deploy.js

```diff
@@ -80,6 +80,11 @@ function parseArgs() {
 }
 
 function spawnOrFail(command, args, options) {
+  if (options) {
```

Comment:
How about assigning since `shell` will always be `true`?

```
options = {
  ...options,
  shell: true
};
```

Reply:
Nice. Updated